programmers/5일차/제일-작은수-제거하기.js

```diff
@@ -0,0 +1,27 @@
+function solution(arr) {
+    const minNum = Math.min.apply(null,arr);
+    const deleteMinArr = arr.filter(element => element !== minNum);
+    console.log(Math.min(arr))
+    if (deleteMinArr.length === 0 || deleteMinArr.length === 1) {
```

Comment:
```suggestion
    if (!deleteMinArr.length) {
```
여기 서는  만약  최솟값이 배열에 없다면  filter함수에 의해서 무조건  빈 배열이 리턴 될 거에요  
따라서  배열 길이가 1이냐 0이냐 조건이 아닌  그냥  배열의 길이가 0인지 조건 하나만 넣어주어도 된답니다